app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -166,8 +164,8 @@ private void showNoPodcastFoundError() {
      * Displays a progress indicator.
      */
     private void setLoadingLayout() {
-        progressBar.setVisibility(View.VISIBLE);
-        findViewById(R.id.feedDisplay).setVisibility(View.GONE);
+        binding.progressBar.setVisibility(View.VISIBLE);
+        binding.feedDisplay.setVisibility(View.GONE);
```

Comment:
`feedDisplay` is a strange name. I had to look at the xml file to find out what it actually is. How do you feel about `feedDisplayContainer`? I know it was like that before and the guidelines we have come up with say that it is not necessary. How do you feel about this? Should we update the guidelines?